docs/release-notes/.Language/preview.md

```diff
@@ -13,5 +13,6 @@
 ### Fixed
 
 * Warn on uppercase identifiers in patterns. ([PR #15816](https://github.com/dotnet/fsharp/pull/15816))
+* Fix nullness warnings when casting non-nullable values to `IEquatable<T>` to match C# behavior. ([Issue #18759](https://github.com/dotnet/fsharp/issues/18759))
```

Comment:
Remove the entry here (this file is for language changes) and put it here: docs/release-notes/.FSharp.Compiler.Service/10.0.100.md